include/kokkos-utils/timer/Event.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef KOKKOS_UTILS_TIMER_EVENT_HPP
+#define KOKKOS_UTILS_TIMER_EVENT_HPP
+
+#include <chrono>
+
+#include "Kokkos_Macros.hpp"
```

Comment:
Why ?

Reply:
I was thinking of the macros `KOKKOS_ENABLE_CUDA` and `KOKKOS_ENABLE_HIP`. But may be better to leave it up to the user that kokkos includes are included before this header. So I deleted it.